var/spack/repos/builtin/packages/py-tensorflow-addons/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTensorflowAddons(PythonPackage):
+    """TensorFlow Addons."""
+
+    homepage = "https://pypi.org/project/tensorflow-addons/"
+    url = "https://github.com/tensorflow/addons/archive/refs/tags/v0.13.0.tar.gz"
+
+    version('0.13.0', sha256='5a8c33ecef5a3daca7f6b27c4b4e3a2badfe55f55a13e97066ee2e03ef98fab0')
+
+    depends_on('py-typeguard@2.7:', type=('run'))
```

Comment:
@pafonta : If I am not mistaken, this needs run dependency with `py-tensorflow` as well?

Reply:
Done in 729204aea9.